### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by bryce-mcmath**
The new v4.0.0 of the common library has breaking changes for most of the apps that use it, so I'm instead going to merge in the accessibility work to the most recent working v3.3.5.

### CODE REVIEW

File: projects/common/lib/components/address/address.component.html [hunk 1]
```diff
@@ -38,6 +38,23 @@
...
+      (ngModelChange)="setStreetAddress($event)"
+      [required]="isRequired"
+      [disabled]="readOnlyFields.address"
+      [maxlength]="fieldMaxLengths.address"
+      [serviceUrl]="addressServiceUrl"
+      (select)="selectSuggestedAddress($event)"
+      [populateAddressOnSelect]="true"
+      commonValidateStreet>
+    </common-address-validator>
+  </div>
```
Comment (harrymaynard-maximus) [inline projects/common/lib/components/address/address.component.html:57]: I think this addition was due to a bad merge. The `common-address-validator` component is already used near the top of this file. The `common-address` component is used throughout MSP (contact pages), please make sure it's not rendering the component twice.

**Review by harrymaynard-maximus — CHANGES_REQUESTED** [review]

File: projects/common/lib/components/address/address.component.html (hunk 1 above)
Comment (bryce-mcmath) [reply projects/common/lib/components/address/address.component.html]: Removed the duplication. Good eye

File: projects/common/lib/components/province/province.component.ts [hunk 2]
```diff
@@ -80,17 +80,22 @@ export class ProvinceComponent extends AbstractFormControl implements OnInit {
     super.ngOnInit();
   }
 
-  onValueChange( value: any ) {
-    if ( value !== this.province ) {
-      this._onChange( value );
-      this.valueChange.emit( value );
-      this.province = value;
+  provinceChange( event: any ) {
```
Comment (stephanieharrison-maximus) [inline projects/common/lib/components/province/province.component.ts:83]: Lines 83-98 here seem a little repetitive. Is there a better way to accomplish this?

**Review by harrymaynard-maximus — APPROVED** [review]

**Review by cyrilbravo — APPROVED** [review]
Accessibility changes looks good to me. Nice work!

**Review by Ian-c-cameron — APPROVED** [review]

**Pull request merged**